app/assets/locales/messages_ccodk_default.txt

```diff
@@ -304,6 +304,8 @@ demo.mode.feature.unavailable=Sorry, this app uses a feature that is not availab
 
 select.search.label=Search:
 select.search.status=${0} of ${1} results for your search: "${2}"
+select.callout.search.status=${0} of ${1} results for callout result

```

Comment:
Hm, is this UX pass supposed to happen before or after it gets merged? The reason I ask is because if it's before, then I'm going to assume that this PR won't get into 2.28 anyway, and I'm going to review something else for now instead..